src/validateEmail.test.js

```diff
@@ -8,11 +8,46 @@ describe(`Function 'validateEmail':`, () => {
   });
 
   it(`should return boolean`, () => {
+    expect(typeof validateEmail('annA12.an$d@gmail.com')).toBe('boolean');
+  });
+
+  it(`should return 'true' for the valid email an12.a@gmail.com`, () => {
+    expect(validateEmail('an12.a@gmail.com')).toBeTruthy();
+  });
+
+  it(`should return 'true' for the valid email
+  with digits an12345sa@gmail.com`, () => {
+    expect(validateEmail('an12345sa.a@gmail.com')).toBeTruthy();
+  });
+
+  it(`should return 'false' for the email
+  with double dots an..45sa@gmail.com`, () => {
+    expect(validateEmail('an..45sa@gmail.com')).toBeFalsy();
+  });
+
+  it(`should return 'false' for the email
+  with double dots an..45sa@gmail.com`, () => {
+    expect(validateEmail('an..45sa@gmail.com')).toBeFalsy();
+  });
 
+  it(`should return 'false' for the email
+  which start with dots .45sa@gmail.com`, () => {
+    expect(validateEmail('.45sa@gmail.com')).toBeFalsy();
   });
 
-  it(`should return 'true' for the valid email...`, () => {
+  it(`should return 'false' for the email
+  without @ 45sagmail.com`, () => {
+    expect(validateEmail('45sagmail.com')).toBeFalsy();
+  });
+
+  it(`should return 'false' for the email
+  without @@ 45sa@@gmail.com`, () => {
+    expect(validateEmail('45sa@@gmail.com')).toBeFalsy();
+  });
 
+  it(`should return 'false' for the email
+  with cyrillic letters Фрaf45dd.com`, () => {
+    expect(validateEmail('Фрaf45dd.com')).toBeFalsy();
```

Comment:
maybe you will recive true for the email with cyrillic, if it have @?

Reply:
corrected